src/config.js

```diff
@@ -1,18 +1,66 @@
 require('dotenv').config();
 const csvParser = require('./utils/csv-parser');
 
-const DISCORD_CONFIG = {
+const DISCORD_SERVER = {
   discordToken: process.env.DISCORD_TOKEN,
   discordClientId: process.env.DISCORD_CLIENT_ID,
   discordGuildId: process.env.DISCORD_GUILD_ID,
   discordQARoleId: process.env.DISCORD_QA_ROLE_ID,
   discordQAChannelName: process.env.DISCORD_QA_CHANNEL_ID,
   discordAdminPointRequestChannel: process.env.ADMIN_POINT_REQUEST_CHANNEL,
   discordAdminTagId: process.env.ADMINISTRATOR_TAG_ID,
+  botLanguage: process.env.DISCORD_LANGUAGE || 'en',
+  cronTimes: {
+    messageTimes: process.env.SCHEDULED_MESSAGES
+      ? csvParser(process.env.SCHEDULED_MESSAGES)
+      : [],
+    timeZone: process.env.TIME_ZONE,
+  },
+  VOTE_POINTS: {
+    ANSWERS: [
+      {
+        text: '1',
+        emoji: '🥇',
+      },
+      {
+        text: '2',
+        emoji: '🥈',
+      },
+      {
+        text: '3',
+        emoji: '🥉',
+      },
+      {
+        text: '4',
+        emoji: '4️⃣',
+      },
+      {
+        text: '5',
+        emoji: '5️⃣',
+      },
+      {
+        text: '6',
+        emoji: '6️⃣',
+      },
+      {
+        text: '7',
+        emoji: '7️⃣',
+      },
+      {
+        text: '8',
+        emoji: '🎱',
+      },
+    ],
+    TAG_IDS: {
+      taskCompletedTagId:
+        process.env.TASK_COMPLETED_TAG_ID || '1203085046769262592',
+      addPointTagId: process.env.ADD_POINT_TAG_ID || '1258801833191669802',
+      boostedPointTagId:
+        process.env.ADD_BOOSTED_POINT_TAG_ID || '1263873487953592381',
+    },
+  },
```

Comment:
I think it could be more atomic, in this group there are various variables that are not group in features (or it is not a visual separation), I think it's still hard to read for me and understand what vars are dependencies of what feature.

My proposal is to separate the dependencies of all the features in its own object, here is an example:

```suggestion
const QA_MENTION = {
  discordQARoleId: process.env.DISCORD_QA_ROLE_ID,
  discordQAChannelName: process.env.DISCORD_QA_CHANNEL_ID,
}

const REQUEST_POINT = {
   discordAdminPointRequestChannel: process.env.ADMIN_POINT_REQUEST_CHANNEL,
   discordAdminTagId: process.env.ADMINISTRATOR_TAG_ID,
}

const SCHEDULE_MESSAGES = {
  messageTimes: process.env.SCHEDULED_MESSAGES
    ? csvParser(process.env.SCHEDULED_MESSAGES)
    : [],
  timeZone: process.env.TIME_ZONE,
  // Here you can add the time zones that are below, since it is used here. Github doesnt let me do it in the suggestion
}

const VOTE_POINTS = {
    ANSWERS: [
      {
        text: '1',
        emoji: '🥇',
      },
      {
        text: '2',
        emoji: '🥈',
      },
      {
        text: '3',
        emoji: '🥉',
      },
      {
        text: '4',
        emoji: '4️⃣',
      },
      {
        text: '5',
        emoji: '5️⃣',
      },
      {
        text: '6',
        emoji: '6️⃣',
      },
      {
        text: '7',
        emoji: '7️⃣',
      },
      {
        text: '8',
        emoji: '🎱',
      },
    ],
    TAG_IDS: {
      taskCompletedTagId:
        process.env.TASK_COMPLETED_TAG_ID || '1203085046769262592',
      addPointTagId: process.env.ADD_POINT_TAG_ID || '1258801833191669802',
      boostedPointTagId:
        process.env.ADD_BOOSTED_POINT_TAG_ID || '1263873487953592381',
    },
  },
}

const DISCORD_SERVER = {
  discordToken: process.env.DISCORD_TOKEN,
  discordClientId: process.env.DISCORD_CLIENT_ID,
  discordGuildId: process.env.DISCORD_GUILD_ID,
  botLanguage: process.env.DISCORD_LANGUAGE || 'en'
```

If there are two features that uses the same .env I think it can be re-assigned in the two objects, it's better to be very clear what .env is used in what feature, what do you think 😄 ?